conftest.py

```diff
@@ -122,3 +123,36 @@ def pytest_configure(config):
     #
     # Somehow the default database is always in memory, though.
     settings.DATABASES['transient']['TEST_NAME'] = ':memory:'
+
+
+@pytest.yield_fixture()
+def installed_packages_catalog():
```

Comment:
Not a big fan of such a big mocking feature: the more we mock, the higher the risks of having falsy tests.
Also I generally tend to use mock for I/O (that we cannot reproduce during tests), but